### DISCUSSION TITLE
Fill Color Separation Check

### DISCUSSION THREAD

**Discussion by JParks123** [opentoonz/opentoonz#6362] [Ideas]
### Is your feature request related to a problem? Please describe

 when using the Fill-Check and an image has its gaps closed it becomes entirely filled with the same color but this leaves out the different sections inside of the image so you wont know if the gaps are closed.

![image](https://github.com/user-attachments/assets/47568799-db3e-4f10-9fbe-1871e4fdd7f3)

### Describe the solution you'd like

this Fill-Check update would make each fill section an independent color as to differentiate the gap closed sections like in Cacani.

![image](https://github.com/user-attachments/assets/7495b56b-d25c-4e47-bafc-21b19efe9beb)

I do think the ordinary all solid fill-check still has its purposes as well so this color separation check should be a new check in itself.

**Comment by RodneyBaker** [comment]
Not implemented yet as either a visual check or as FX but the beginnings of movement in that direction can be seen in the addition of Lazy Brush Naru FX which is available in current nightly release.  REF:  #6040
